WordPress/src/main/java/org/wordpress/android/ui/accounts/SiteCreationActivity.java

```diff
@@ -183,7 +183,7 @@ public void withSiteDetails(String siteTitle, String siteTagline) {
         mSiteTitle = siteTitle;
         mSiteTagline = siteTagline;
 
-        SiteCreationDomainFragment fragment = SiteCreationDomainFragment.newInstance(mUsername);
```

Comment:
Now that `mUsername` is no longer used here, we can remove it from the class completely as well as the `ARG_USERNAME` constant since they are only being used to assign the `mUsername` value [here](https://github.com/wordpress-mobile/WordPress-Android/blob/583ac592887704b88585ebf1095d933cccf9764c/WordPress/src/main/java/org/wordpress/android/ui/accounts/SiteCreationActivity.java#L50), but never accessed.

I know it's not part of these changes, but the `mSiteDomain` variable falls into the same category; it's assigned, but never accessed and can be removed.